samcli/commands/translate/translate_utils.py

```diff
@@ -0,0 +1,31 @@
+"""
+Utils for performing Sam translates
+"""
+import os
+import logging
+import click
+
+
+LOG = logging.getLogger(__name__)
+
+
+def read_sam_file(template):
```

Comment:
I found a similar function in `samcli/commands/_utils/template.py` maybe we can just re-use that one.

Reply:
switched to using that one

Reply:
Do we still need this file?

Reply:
File removed